test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -357,6 +357,15 @@ TEST(QuicPlatformTest, QuicDLog) {
 
 #undef VALUE_BY_COMPILE_MODE
 
+TEST(QuicPlatformTest, QuicDCHECK) {
+  CHECK(1 == 1);
+  CHECK(1 == 1) << " 1 == 1 is forever true.";
+  EXPECT_DEBUG_DEATH({ CHECK(false) << " Supposed to fail in debug mode."; },
```

Comment:
Should be DCHECK since CHECK fails in release mode. Have you tried to test it with -c opt?

Reply:
oh, yes. Added both CHECK and DCHECK here.